src/components/ListBox/component.js

```diff
@@ -0,0 +1,57 @@
+import { h, Component, Fragment } from "preact";
+import {
+  ListboxInput,
+  ListboxButton,
+  ListboxPopover,
+  ListboxList,
+  ListboxOption,
+} from "@reach/listbox";
+import "@reach/listbox/styles.css";
+import { Icon } from "@Components/Icon";
+import PropTypes from 'prop-types';
+
+export default class ListBox extends Component {
+  render() {
+    const { defaultValue, placeholder, options, onChange } = this.props;
+    
+    /**
+     * If no options yet, we don't want to render the component.
+     */
+    if (!options) {
+      return;
+    }
+
+    return (
+      <ListboxInput className="sn-listbox" defaultValue={defaultValue} onChange={onChange}>
+        {({ value, valueLabel, isExpanded }) => (
+          <Fragment>
```

Comment:
You can use `<></>` instead of `<Fragment></Fragment>`, personally I think it better communicates the fact that there is no actual element being created. Let me know if you agree!

Reply:
I didn't use the short syntax `<></>` because it throws `React not defined` error:

![image](https://user-images.githubusercontent.com/5891646/108584139-4a1f3a00-7315-11eb-910a-980175a03acc.png)

The docs at https://preactjs.com/guide/v10/components/#fragments mention it's usage.

I'll take a look at what is causing the error.

Reply:
I've gotten it to work on the web repo which uses preact, if you'd like to compare implementations :)

Reply:
I've checked the implementation on `web`, but I can't tell what's the issue. I will leave it as-is for now